libs/logging/src/types.ts

```diff
@@ -5,8 +5,9 @@ import {
 } from '@votingworks/types';
 import { z } from 'zod';
 import { DeviceType } from '@votingworks/cdf-types-election-event-logging';
-import { LogEventId, LogEventType } from '.';
+import { LogEventId } from './log_event_ids';
```

Comment:
There was a weird bug without these changes where the z.nativeEnum call was created a enum parser with no possible enum values defined. For some reason this only happened when importing to election-manager and not when the tests in libs/logging where running. I'm guessing it was some kind of circular dependency problem.